### ISSUE TITLE
Suggestion for `try` functions

### ISSUE THREAD

**Issue by Happypig375** [vzarytovskii/FSharp.Core.Faster#2]
I propose we get rid of the `try` naming altogether and return valueoption by default. Existing throwing functions should have a prefix like `unsafe` instead: `tryHead` -> `head`, `head` -> `unsafeHead`.

**Comment by vzarytovskii** [comment]
> I propose we get rid of the `try` naming altogether and return valueoption by default. Existing throwing functions should have a prefix like `unsafe` instead: `tryHead` -> `head`, `head` -> `unsafeHead`.

I want this library to be a drop-in replacement, meaning I intend to leave try functions in place.

We can change behaviour of normal ones too

**Comment by Happypig375** [comment]
We can still include the try prefix functions for back compat then. But we can add unsafe functions and have the unprefixed functions default to try instead.

**Comment by vzarytovskii** [comment]
> We can still include the try prefix functions for back compat then. But we can add unsafe functions and have the unprefixed functions default to try instead.

Yeah, need to emit proper messages/obsolete/compiler warnings and have a doc, since won't be a drop-in replacement anymore (for non-try variant).

**Comment by Happypig375** [comment]
Also - rename `Option.get` to `Option.getOrThrow`
